From c52b9a098bcc41072ddd76546b986ba6d467e835 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Sun, 24 Apr 2016 17:27:26 +0300
Subject: [PATCH 19/46] ARM: dts: imx7d: sbc-imx7: add PCIe support

Enable PCIe support. Set internal PLL as a PCIe PHY reference clock source.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-sbc-imx7.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sbc-imx7.dts b/arch/arm/boot/dts/imx7d-sbc-imx7.dts
index b50714f..bda9382 100644
--- a/arch/arm/boot/dts/imx7d-sbc-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-sbc-imx7.dts
@@ -203,6 +203,14 @@
 	};
 };
 
+&pcie {
+	pinctrl-names = "default";
+	depends = "/soc/aips-bus@30800000/i2c@30a50000/pca9555_sb@20";
+	reset-gpio = <&pca9555_sb 1 GPIO_ACTIVE_LOW>;
+	phy-ref-clk = <1>;
+	status = "okay";
+};
+
 &pwm2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_pwm2>;
-- 
1.9.1

